build.config

```diff
@@ -86,7 +86,7 @@
   "module": {
     "always": ["buffer", "console", "events", "fs", "module", "timers"],
     "optional" : ["assert", "dgram", "dns", "gpio", "http", "i2c", "net",
-                  "pwm", "stream", "testdriver"],
+                  "pwm", "stream", "testdriver", "ble-central"],
```

Comment:
module order should be by alphabet order so that ble-central is goining after assert